code/game/mecha/mecha.dm

```diff
@@ -889,6 +889,9 @@
 	if(HAS_TRAIT(user, TRAIT_PRIMITIVE)) //no lavalizards either.
 		to_chat(user, "<span class='warning'>The knowledge to use this device eludes you!</span>")
 		return
+	if(HAS_TRAIT(user, TRAIT_DUMB)) //braindead idiots don't get in the mech
```

Comment:
this line of code is offensive

Reply:
This snippet of code is a port from girlystation

https://github.com/tgstation/tgstation/pull/45017

Reply:
I can't allow this information to get out. I'm sorry, I must now close this.